### DISCUSSION TITLE
Can't differentiate between inactive and active button for the new wedge buttons

### DISCUSSION THREAD

**Discussion by fyyrraaa** [ppy/osu#18046] [Q&A] [answered]
I had a look at the in-progress first-run setup guide out of curiosity, and it seems like for me the perceived button colour difference between an active button and an inactive one is somewhat hard to tell.

(EDIT: what I meant to show is the difference between an inactive back button and an active one, my bad!)

![osu! 2022-05-01 7_44_58 PM](https://user-images.githubusercontent.com/3055647/166174990-53deb06e-daf1-417a-83cb-d4c94292b927.png)

![Import beatmaps page](https://user-images.githubusercontent.com/3055647/166174740-7b784b99-2f3f-4ddf-b359-48430a56f6c5.png)

While in hindsight it would be clear as I go through the process, it might be better to have more contrast between the inactive and active colour for the button, or just hiding the back button altogether in the welcome page.

# Logs (just in case)
[runtime.log](https://github.com/ppy/osu/files/8600396/runtime.log)
[updater.log](https://github.com/ppy/osu/files/8600397/updater.log)
[database.log](https://github.com/ppy/osu/files/8600398/database.log)
[network.log](https://github.com/ppy/osu/files/8600399/network.log)
[performance.log](https://github.com/ppy/osu/files/8600400/performance.log)

(note that the logs are taken while the game is still running)

## Specs
- MacBook Pro 16" (2019)
    - Intel Core i9-9880H
    - 32 GB RAM
    - AMD Radeon Pro 5500M (8 GB VRAM)
        - Driver Version
21.30.33.01-211028a-373142E-RadeonSoftware
- Windows 11 Enterprise Insider Preview (22H1, build 22610.1)

**Comment by peppy** [comment, marked as answer]
Can you please explain how this affects your usability? Hover effects are usually for visual flair and nothing more. Do you have a certain input device or disability where this decreases your ability to find buttons?

**Reply by fyyrraaa** [reply]
Sorry, I should've been clearer.

What I meant is for the back button, when it's inactive it looks a bit too similar with when it's active. I'll annotate the screenshot next time.

**Reply by peppy** [reply]
Aha, understood. I thought it was referring to the hover states.

The inactive state can potentially be adjusted slightly, I agree it's a but subtle.

**Reply by fyyrraaa** [reply]
Would it be possible for the welcome page to just have the one big next button? Or would that not work?

**Reply by peppy** [reply]
No. Because you can't go back then.

**Reply by fyyrraaa** [reply]
Fair enough.

**Reply by frenzibyte** [reply]
Should be resolved in https://github.com/ppy/osu/pull/18053
